scala/scala.bzl

```diff
@@ -682,9 +682,17 @@ def _scala_test_impl(ctx):
 
     transitive_rjars += [ctx.outputs.jar]
 
+    # output report test duration
```

Comment:
Any chance you can extract this block into a "flags" method? This will decrease the likelihood of us having a collision.

Reply:
absolutely. I eta on that is probably within the next day or so